File: Getting-Practice-with-Python-and-Javascript/4- Functions/Javascript/Excercises/program-3.js
// 3.  Escribe una función que tome una lista de números y devuelva el mayor número en la lista.


// Solution: 

let majorNum = (list) => {
    let counter = 0;
    for (let i = 0; i < list.length; i++) {
        if (list[i] > counter) {
            counter = list[i]
        }

    }
    document.write(counter)
}


majorNum([10, 1, 2, 3, 4, 5, 6])

File: Getting-Practice-with-Python-and-Javascript/4- Functions/Python/Excercises/program-3.py
# 3.  Escribe una función que tome una lista de números y devuelva el mayor número en la lista.


# Solution:  
def majorNum(list):
    counter = 0
    for i in list:
        if counter < i:
            counter = i 
    print(counter)


majorNum([1,2,300,4,50,66])